### PR TITLE
Added new corrections tasks to main.py task list

### PR DESCRIPTION
**Pull request by coderdj**
Should be clear why the new corrections could not work without this step. Whoops.

### CODE REVIEW

File: cax/main.py [hunk 1]
```diff
@@ -101,6 +101,10 @@ def main():
         corrections.AddElectronLifetime(),  # Add electron lifetime to run, which is just a function of calendar time
         corrections.AddGains(), #  Adds gains to a run, where this is computed using slow control information
         corrections.AddDriftVelocity(), #  Adds drift velocity to the run, also computed from slow control info
+	corrections.SetS2xyMap(),
```
Comment (pdeperio) [inline cax/main.py]: missing space?

Comment (coderdj) [inline cax/main.py]: Hold on. Web editor doesn't allow tab.

**Comment by pdeperio** [comment]
I got a lot errors with your tabs before, use spaces please

File: cax/main.py (hunk 1 above)
Comment (coderdj) [inline cax/main.py]: Did you fix it? Looks fine to me.

huh

**Comment by coderdj** [comment]
OK fixed now. I'll try to use spaces when using web editor because it seems to have problems with tabs.

**Comment by pdeperio** [comment]
merging, hopefully deployHQ works.  can someone @lucrlom check and restart can daemon? and  after purging pax v6.6.5 ?

**Comment by lucrlom** [comment]
I have to purge all 6.6.5? ok. 
I stopped cax, now I will reinstall manually on 6.6.5 the new version. Deploy work only on pax_head

**Comment by pdeperio** [comment]
this merge should deploy to both head and latest pax environment

**Comment by lucrlom** [comment]
Hi guys, 
It seems to be a problem with the processed filespax_v6.6.5, they are not recorded on RunDB, therefore cax is not able to delete it.

**Pull request merged**